index.html

```diff
@@ -51,10 +54,10 @@
         </div>
     </div>
     <div class="header-bottom row">
-        <div class="col-mobile-100">
-            <img class="col-large-tablet-hide col-desktop-hide" src="image/mobile-header.jpg" alt="">
+        <div class="max col-mobile-100 col-large-tablet-hide col-desktop-hide">
```

Comment:
do czego sluzy klasa max? Czy mozesz zamienic jej nazwe zgodnie z terminologia BEM?

B - blok
E - element
M - modyfikator

W tym wypadku klasa class="max" jest IMO modyfikatorem, jednak nie okresla jasno jaki blok-element modyfikuje. To powinno wygladac np.
header--max => tutaj odnosi sie do calego bloku
header_link--max => tutaj odnosi sie do elementu link w bloku header